apps/web-giddh/src/assets/locale/invoice/mr.json

```diff
@@ -142,6 +142,15 @@
 		"na": "या व्यवहारासाठी ई-इनव्हॉईस निर्मिती लागू नाही.",
         "giddh_invoice_not_cancelled": "ई-इनव्हॉईज्ड व्यवहार आयआरपीमध्ये रद्द केला गेला आहे, परंतु गिद्धमध्ये नाही. त्याच्याशी संबंधित आयआरएन यापुढे वैध नाही."
     },
+    "e_invoice_statuses_label": {
```

Comment:
These status will come from GST portal so will be in English for all the language